sql/core/src/test/resources/sql-tests/results/ansi/string-functions.sql.out

```diff
@@ -1597,3 +1597,20 @@ org.apache.spark.sql.AnalysisException
     "invalidValue" : "invalidformat"
   }
 }
+
+
+-- !query
+SELECT to_binary(col1, col2) from values ('abc', 'utf-8') as data(col1, col2)
+-- !query schema
+struct<>
+-- !query output
+org.apache.spark.sql.AnalysisException
+The 'format' parameter of function 'to_binary' needs to be a string literal.; line 1 pos 7
```

Comment:
When you move the exception from constructor, you should see `AnalysisException` w/ an error class.